Makefile

```diff
@@ -6,19 +6,29 @@ ifeq ($(CONTAINER_CMD),)
 	CONTAINER_CMD:=$(shell docker version >/dev/null 2>&1 && echo docker)
 endif
 
+BUILD_TRIGGER ?=
```

Comment:
Instead of having conditionals in the makefile, I suggest we add new targets. See below. I also find this variable name confusing as "trigger" makes it sound like it should be activating something extra.